docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -108,16 +108,21 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 Configuration changes require stopping the Alluxio Docker images, then re-launching
 them with the new configuration.
 
-To set an Alluxio configuration property, convert it to an environment variable by uppercasing
-and replacing periods with underscores. For example, `alluxio.master.hostname` converts to
-`ALLUXIO_MASTER_HOSTNAME`. You can then set the environment variable for the image with
-`-e PROPERTY=value`. Alluxio configuration values will be copied to `conf/alluxio-site.properties`
+To set an Alluxio configuration property, add it to the alluxio java options environment variable with
+
```

Comment:
is this newline intentional?

Reply:
I always leave a newline before multi-line code blocks, I think omitting the newline might cause the code block to be combined into the previous text line in some flavors of markdown